src/modules/module_28500.c

```diff
@@ -72,6 +72,20 @@ int module_hash_decode (MAYBE_UNUSED const hashconfig_t *hashconfig, MAYBE_UNUSE
   u8 * digest = (u8 *) digest_buf;
   u8 * pubkey = hccalloc(PUBKEY_MAXLE,1);
```

Comment:
please do the memory allocation (`hccalloc ()`) after the tokenizer accecpted the "hash", otherwise we could have a memory leak